deepfence_frontend/apps/dashboard/src/features/settings/pages/UserAuditLogs.tsx

```diff
@@ -119,7 +119,7 @@ const UserAuditLogs = () => {
                       data={logs}
                       columns={columns}
                       enablePagination
-                      pageSize={30}
+                      pageSize={10}
```

Comment:
keep page size 30. check column widths to make sure table looks good on 720p

Reply:
kept page size 30.

Reply:
for 720px showing 7 columns is like squished with less real estate but all the columns looks find with ellipsis wherever applicable with current width values